### PR TITLE
Add testnet pipeline for prebuilt images

### PR DESCRIPTION
**Pull request by pgarg66**
- It'll speed up testnet testing for released images

#### Problem
Testnet automation builds a new snap for performance testing. For a release perf testing, creating a snap is not required. The release is already archived on cloud. It'll speed up testing if the prebuilt images are used.

#### Summary of Changes
Added a new pipeline for using prebuilt images

Fixes #

### CODE REVIEW

File: ci/testnet-automation.sh [hunk 1]
```diff
@@ -28,7 +29,11 @@ launchTestnet() {
   net/init-metrics.sh -e
 
   echo --- start "$nodeCount" node test
-  net/net.sh start -o noValidatorSanity -S solana_*.snap
+  if [[ -n "$PREBUILT" ]]; then
```
Comment (mvines) [inline ci/testnet-automation.sh]: where does PREBUILT get set?

Comment (pgarg66) [reply ci/testnet-automation.sh]: It's pipeline env variable. I added a new pipeline (solana-testnet-prebuilt)

File: ci/solana-testnet-prebuilt.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+steps:
```
Comment (mvines) [inline ci/solana-testnet-prebuilt.yml]: let's merge this with solana-testnet.yml, but with:

```diff
-command: "ci/snap.sh"
+command: "test -n $PREBUILT || ci/snap.sh"
```

Comment (pgarg66) [reply ci/solana-testnet-prebuilt.yml]: 👍

File: ci/testnet-automation.sh [hunk 3]
```diff
@@ -28,7 +29,11 @@ launchTestnet() {
   net/init-metrics.sh -e
 
   echo --- start "$nodeCount" node test
-  net/net.sh start -o noValidatorSanity -S solana_*.snap
+  if [[ -n $PREBUILT ]]; then
```
Comment (mvines) [inline ci/testnet-automation.sh]: Can we use a more descriptive name here: PREBUILT -> USE_PREBUILT_CHANNEL_TARBALL perhaps

Comment (pgarg66) [reply ci/testnet-automation.sh]: 👍

**Pull request merged**